src/Activators/FileActivator.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+namespace Nwidart\Modules\Activators;
+
+use Illuminate\Cache\CacheManager;
+use Illuminate\Config\Repository as Config;
+use Illuminate\Container\Container;
+use Illuminate\Filesystem\Filesystem;
+use Nwidart\Modules\Contracts\ActivatorInterface;
+use Nwidart\Modules\Module;
+
+class FileActivator implements ActivatorInterface
+{
+	/**
```

Comment:
Why are fields/methods protected instead of private? Could you make all `protected` fields and methods private?